pyproject.toml

```diff
@@ -56,6 +56,7 @@ source = "vcs"
 
 [tool.hatch.version.raw-options]
 local_scheme = "dirty-tag"
+version_scheme = "calver-by-date"
```

Comment:
Whoops, this was accidentally added back (removed in https://github.com/conda/conda/pull/12708)